# SUMMARY OF FINDINGS: DATA ANALYSIS FOR CIR SHARK REPORT

## 1. THE DATA

In [1]:
#Import Data
CMAR = read.csv("/Users/sarahblaskey/Documents/Stats/SharkTradeCMAR.csv")
CR = read.csv("/Users/sarahblaskey/Documents/Stats/SharkExportCRCLEAN.csv",as.is=TRUE)
offload = read.csv("/Users/sarahblaskey/Documents/Stats/INCOPESCA.csv",as.is=TRUE)

### 1.1 CMAR data 

The data is called "CMAR" because it includes shark trade data from all of the countries that have coastline along the Eastern Tropical Pacific Marine Corridor (known as CMAR), as well as the neighboring countries (Nicaragua, Guatemala, Honduras, and El Salvador) which are not traditionally listed as CMAR but often receive offloads from the area. This data is considered an official source of trade data, and allows us to see which countries are most prominent in the regional shark trade.

*Data from the online repository of the United Nations Statistics Division (UNSD). The data is voluntarily submitted by participating countries. This type of trade data is collected from bills of lading, which record international commodity movements, categorized by hs code. Bills of Lading are used by insurance companies to insure cargo, amongst other things. While in some cases there is reason for countries to ommit data from the UNSD, that is not usually an issue with shark products. The more difficult part is tracking what products are actually shark and what are not because of the wide range of HS codes (universal commodity codes) that are used for this commodity. Different countries will use different HS codes for shark products.*

I collected the data from the UN COMTRADE online database. https://comtrade.un.org/ The following criteria were selected: 

Type of Product: Goods ; Frequency: Annual ; Classification: HS-"As Reported" ; Periods(year): 2016,2015,2014,2013,2012,2011 ; Reporters: El Salvador, Guatemala, Honduras, Nicaragua, Costa Rica, Panama, Colombia, Ecuador ; Partners: All ; Trade Flows: All Commodity Code:
030265: 030281; 030292; 030375; 030381; 030392; 030447; 030456; 030488; 030496; 030571; 160418
(Commodity Codes selected by typing in "Shark" and selecting all applicable)

I removed duplicates based on TradeValue because some of the shipments with "World" as partner were duplicates of other shipments already listed.


In [2]:
#1.1.1 An example of what the CMAR data looks like
head(CMAR)

Classification Year Period Period.Desc. Aggregate.Level Is.Leaf.Code
1 H3             2011 2011   2011         6               1           
2 H3             2011 2011   2011         6               1           
3 H3             2011 2011   2011         6               1           
4 H3             2011 2011   2011         6               1           
5 H3             2011 2011   2011         6               1           
6 H3             2011 2011   2011         6               1           
  Trade.Flow.Code TradeFlow ReporterCode Reporter   ⋯ Qty Alt.Qty.Unit.Code
1 2               Export    558          Nicaragua  ⋯ NA  NA               
2 2               Export    188          Costa Rica ⋯ NA  NA               
3 2               Export    218          Ecuador    ⋯ NA  NA               
4 1               Import    558          Nicaragua  ⋯ NA  NA               
5 2               Export    188          Costa Rica ⋯ NA  NA               
6 2               Export    558          Nicaragua  ⋯ NA  NA               
  Alt.Qty.Unit Alt.Qty Netweight Gross.weight..kg. TradeValue CIF
1 105774       NA      105774    NA                98534      NA 
2   7594       NA        7594    NA                46136      NA 
3   5566       NA        5566    NA                40770      NA 
4  65370       NA       65370    NA                39407      NA 
5   5320       NA        5320    NA                37240      NA 
6  45560       NA       45560    NA                34170      NA 
  FOB.Trade.Value..US.. Flag
1 NA                    0   
2 NA                    0   
3 NA                    0   
4 NA                    0   
5 NA                    0   
6 NA                    0

### 1.2 CR data 

This is a dataset of shark product exports from Costa Rica between Jan.2011 and Aug.2017 (the most recent data available). The data is based on bills of lading. 

*This is an unofficial, but still widely respected aggregate of trade data from Costa Rica by a private Arizona-based company, ImportGenius, which launched in 2007 and has since gained a reputation as a reliable source for trade data amongst shipping companies, investors, and investigators alike.* 

*According to its website: "Import Genius reveals the trading activities of importers and exporters around the world. We give you hard data about your overseas suppliers and domestic competitors. Using shipping databases from Customs agencies in the United States, Latin America, and India, Import Genius can help you find reliable new trading partners, monitor your competitors’ shipments, and keep an eye on your current suppliers and customers. Our team of seasoned import-export veterans, cutting-edge software developers, and customer service professionals has worked for years to build the world's most powerful and accessible database of international trade intelligence." https://www.importgenius.com/*

I collected the data from the importgenius online repository. The following criteria were selected:

Country: Costa Rica ; Search: Export Data ; 1/3/2011 - 8/31/2017
Product: Tiburon; OR Shark; OR Aleta; OR Fins; NOT Atun; OR CARCHARHINUS

After collecting the raw data, I modified it in the following ways:

1) Extracted year from export date column and created new column: Export_Year
2) Run through OpenRefine in order to clean several columns and make it easier to analyze.
    -Add column based on PRODUCT_DESCRIPTION; create new column; ProductDescription2
    -ProductDescription2 cleaned:
    -Key Collision: Fingerprint: accept all suggestions; ngram-fingerprint: Accept all merge and re-cluster
    -Metaphone3 Accept all suggestions with those that are clearly not shark renamed as "delete"
    -Delete all obviously not shark rows by hand
    -Add column based on ProductDescription2 called “PRODUCT”
    -Divide into carne; carnada; aleta; other
    -Add column based on SHIPPER called EXPORTCOMPANY which will serve as original
    -Clean SHIPPER
    -Cluster: 
    -Export csv: SharkExportCRCLEAN.csv
    
There are several problems with this data: 1) It's difficult to search by HS code. Instead I searched by product descriptions with words related to shark products. As a result, it is likely that I missed some shark exports. 2)The data is messy (ie names spelled differently, unrelated products within the data), and requires a lot of cleaning. I attempted to document all editorial decisions made during the aggregating and cleaning process. It is my belief that this data is clean, and an accurate representation of the shark export industry in Costa Rica. If there is a mistake, it is that I did not get all of the relavent data. This data represents a conservative estimate. 3)Product descriptions are more accurate than those provided from UN COMTRADE so it is possible to separate fin exports from meat exports. However, the clasification has to be done by hand and required editorial decisions in order to determine what is a fin and what is meat (or bait, or other) when the description is not clear. 

In [3]:
#1.1.2 An example of what the CR data looks like.
head(CR)

PRODUCT_DESCRIPTION                        ProductDescription2              
1 CARCHARHINUS FALCIFORMIS DRYED SHARK FINS- CARCHARHINUS FALCIFORMIS-        
2 CARCHARHINUS FALCIFORMIS DRYED SHARK FINS- CARCHARHINUS FALCIFORMIS-        
3 CARCHARHINUS FALCIFORMIS DRYED SHARK FINS- CARCHARHINUS FALCIFORMIS-        
4 CARCHARHINUS FALCIFORMIS DRYED SHARK FINS- CARCHARHINUS FALCIFORMIS-        
5 CARCHARHINUS FALCIFORMIS DRYED SHARK FINS- CARCHARHINUS FALCIFORMIS-        
6 PRIONACE GLAUCA DRYED SHARK FINS-          PRIONACE GLAUCA DRYED SHARK FINS-
  PRODUCT SHIPPER               EXPORTCOMPANY                      
1 aleta   INVERSIONES CRUZ Z SA INVERSIONES CRUZ Z SOCIEDAD ANONIMA
2 aleta   INVERSIONES CRUZ Z SA INVERSIONES CRUZ Z SOCIEDAD ANONIMA
3 aleta   INVERSIONES CRUZ Z SA INVERSIONES CRUZ Z SOCIEDAD ANONIMA
4 aleta   INVERSIONES CRUZ Z SA INVERSIONES CRUZ Z SOCIEDAD ANONIMA
5 aleta   INVERSIONES CRUZ Z SA INVERSIONES CRUZ Z SOCIEDAD ANONIMA
6 aleta   INVERSIONES CRUZ Z SA INVERSIONES CRUZ Z SOCIEDAD ANONIMA
  SHIPPER_ADDRESS                                                         
1 DE MARISCOS BAR 25 M. AL NORTE Y 35 MTS. AL OESTE EL COCAL DE PUNTARENAS
2 DE MARISCOS BAR 25 M. AL NORTE Y 35 MTS. AL OESTE EL COCAL DE PUNTARENAS
3 DE MARISCOS BAR 25 M. AL NORTE Y 35 MTS. AL OESTE EL COCAL DE PUNTARENAS
4 DE MARISCOS BAR 25 M. AL NORTE Y 35 MTS. AL OESTE EL COCAL DE PUNTARENAS
5 DE MARISCOS BAR 25 M. AL NORTE Y 35 MTS. AL OESTE EL COCAL DE PUNTARENAS
6 DE MARISCOS BAR 25 M. AL NORTE Y 35 MTS. AL OESTE EL COCAL DE PUNTARENAS
  EXPORT_DATE EXPORT_YEAR HS_CODE    BRAND CIF      NET_WEIGHT PACKAGE       
1 8/28/17     2017        3.0571e+10 SM     5266.72  31.40     BULTOS/PACKAGE
2 8/28/17     2017        3.0571e+10 SM    36120.88 271.30     BULTOS/PACKAGE
3 8/28/17     2017        3.0571e+10 SM    21601.68 199.96     BULTOS/PACKAGE
4 8/28/17     2017        3.0571e+10 SM    20084.99 248.30     BULTOS/PACKAGE
5 8/28/17     2017        3.0571e+10 SM     4350.10 106.10     BULTOS/PACKAGE
6 8/28/17     2017        3.0571e+10 SM     4669.60  89.80     BULTOS/PACKAGE
  X.PACKAGES GROSS_WEIGHT NET_WEIGHT2 TOTAL_INVOICE
1 1          1476         1416.16     107116.3     
2 8          1476         1416.16     107116.3     
3 5          1476         1416.16     107116.3     
4 6          1476         1416.16     107116.3     
5 2          1476         1416.16     107116.3     
6 3          1476         1416.16     107116.3     
  REMARKS                                                                                      
1 GUIA AEREA: 020-5720-3775 , REC 4631, EXPORTADOR: INVERSIONES CRUZ. Z. S.A. CED: 310123731211
2 GUIA AEREA: 020-5720-3775 , REC 4631, EXPORTADOR: INVERSIONES CRUZ. Z. S.A. CED: 310123731211
3 GUIA AEREA: 020-5720-3775 , REC 4631, EXPORTADOR: INVERSIONES CRUZ. Z. S.A. CED: 310123731211
4 GUIA AEREA: 020-5720-3775 , REC 4631, EXPORTADOR: INVERSIONES CRUZ. Z. S.A. CED: 310123731211
5 GUIA AEREA: 020-5720-3775 , REC 4631, EXPORTADOR: INVERSIONES CRUZ. Z. S.A. CED: 310123731211
6 GUIA AEREA: 020-5720-3775 , REC 4631, EXPORTADOR: INVERSIONES CRUZ. Z. S.A. CED: 310123731211
  TRANSPORTATION
1 Aereo         
2 Aereo         
3 Aereo         
4 Aereo         
5 Aereo         
6 Aereo

### 1.3 Offload data 

This is official offload data from all fishing vessels that unloaded fish at any Costa Rican port between 2011 and 2016. 

*The data is collected by inspectors from INCOPESCA, the government agency tasked with regulating the fishing industry in Costa Rica. The inspectors are present for the entire offload, and mark the number and weight of each species offloaded. This data should reflect all of the fish and sharks offloaded in Costa Rica, by local or international vessels in that timeframe, although critics accuse INCOPESCA of underreporting.*

*Costa Rica was the only CMAR country to provide offload data. (Note: every country should hold this data from inspection, and each country in the region was solicited for the information via email to the appropriate institution and failed to provide.)* 

The data is largely unmodified. I added a single row "Species" which is based on the original column "Especies". Through my own judgement and knowledge of fish species, I distiled "Especies" into three categories--Tiburon, Pescado, Other. That distilation is reflected in the "Species" column. Additionally, years 2014 and 2015 have a column "embarcaciones" which detail the vessels that offloaded the product. This was intended to be redacted but they made a mistake on these two years allowing me access to the information.

In [4]:
#1.1.3 An example of what the Offload data looks like.
head(offload)

Fecha    Ano  Bandera    Puerto     Muelle     Solicitud Desembarque   
1 1-Jan-16 2016 Costa Rica Puntarenas Palmarenos 3464      Aleta Adherida
2 1-Jan-16 2016 Costa Rica Puntarenas Palmarenos 3464      Aleta Adherida
3 1-Jan-16 2016 Costa Rica Puntarenas Palmarenos 3464      Aleta Adherida
4 1-Jan-16 2016 Costa Rica Puntarenas Palmarenos 3464      Aleta Adherida
5 1-Jan-16 2016 Costa Rica Puntarenas Palmarenos 3464      Aleta Adherida
6 1-Jan-16 2016 Costa Rica Puntarenas Palmarenos 3464      Aleta Adherida
  Aprobado Especies        Species cuerpos KgCuerpos KgAletas Peso Embarcacion
1 Aprobado Marlin Rosado   pescado 18                         404             
2 Aprobado Vela            pescado  8                         196             
3 Aprobado Atun A Amarilla pescado  3                         82              
4 Aprobado Espada          pescado  2                         37              
5 Aprobado Martillo Rosado tiburon  1      5.5       0.4      5.9             
6 Aprobado Zorro Pelagico  tiburon 29      761       36       797

In [5]:
#packages
library(dplyr)


Attaching package: ‘dplyr’

The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union



## 2. CMAR EXPORTS

In this section, using CMAR data, we will analyze regional trade data in order to determine which CMAR countries are the biggest players in the shark industry. 

### 2.1 Total Shark Exports

In [6]:
#2.1.1 Total weight (Kilo) of potential shark products exported in CMAR countries between 2011 and 2016
#First, subset exports
CMARexport = filter(CMAR, TradeFlow == 'Export')
#Then sum of kilos exported
sum(CMARexport$Netweight)

[1] 26809905

In [7]:
#2.1.2 Total value (USD) of potential shark exports from CMAR countries between 2011 and 2016
sum(CMARexport$TradeValue)
(sum(CMARexport$TradeValue))/6

[1] 67991940

[1] 11331990

26,809,905 kilos of shark product exported in region between 2011 and 2016. $67,991,940 of shark products exported. That's an average of 11,331,990 USD of exports annually.

In [8]:
#2.1.3 Total exports of shark products from CMAR countries by year.
sort(tapply(as.numeric(CMARexport$TradeValue), CMARexport$Year, FUN=sum, na.rm=TRUE))

2011     2012     2013     2014     2015     2016 
 6433500  8192187  9524832 10894550 13689859 19257012

Declared value of shark products has increased (tripled) from CMAR nations between 2011 and 2016.

### 2.2 Ranking of exports by country 

In [9]:
#2.2.1 Total shark export value by country listed from least to most. 
sort(tapply(as.numeric(CMARexport$TradeValue), CMARexport$Reporter, FUN=sum, na.rm=TRUE))


Guatemala   Nicaragua    Colombia El Salvador      Panama     Ecuador 
     177732     1087342     1809871     6095621     6574598    12078697 
 Costa Rica 
   40168079

Between 2011 and 2016, Costa Rica exported 40,168,079 USD worth of shark products.

Costa Rica is the largest exporter of shark products (by declared value), exporting over three times as much as the nearest competitor Ecuador. Panama and El Salvador each export about half of what Ecuador exports.

In [10]:
#2.2.2 Total shark export weight by country listed from least to most
sort(tapply(as.numeric(CMARexport$Netweight), CMARexport$Reporter, FUN=sum, na.rm=TRUE))


Colombia   Guatemala   Nicaragua El Salvador     Ecuador      Panama 
      73422      226456      812595     1452097     4588056     4642673 
 Costa Rica 
   15014606

Costa Rica exported 15,014,606 kilos of shark products between 2011 and 2016. It exports more than three times the weight of the next two countries, Ecuador and Panama. Heavier exports could mean that those countries export more meat in addition to fins. Fins, though high in export value, are very light, making this a less useful metric than declared value of exports.

In [11]:
#2.2.3 Shark exports in price per kilo for each country
sort((tapply(as.numeric(CMARexport$TradeValue), CMARexport$Reporter, FUN=sum, na.rm=TRUE))/(tapply(as.numeric(CMARexport$Netweight), CMARexport$Reporter, FUN=sum, na.rm=TRUE))
)

Guatemala   Nicaragua      Panama     Ecuador  Costa Rica El Salvador 
  0.7848412   1.3381106   1.4161234   2.6326394   2.6752669   4.1978057 
   Colombia 
 24.6502547

The average price per kilo could indicate countries that export a higher ratio of fins to meat. A high price per kilo would imply more fins exported, where are lower price would indicate low quality meat. By that metric, Colombia, and to a lesser extent El Salvador, Costa Rica, and Ecuador export higher ratios of fins.

### 2.3 Costa Rica Exports

In [12]:
#2.3.1 Create Costa Rica subset, average value of shark exports per year, and shark export value per year.
crexports = filter(CMAR, TradeFlow == 'Export', Reporter == 'Costa Rica')
(sum(crexports$TradeValue))/(sum(as.numeric(CMARexport$TradeValue)))
sum(crexports$TradeValue)/6
tapply(as.numeric(crexports$TradeValue), crexports$Year, FUN=sum, na.rm=TRUE)
(tapply(as.numeric(crexports$TradeValue), crexports$Year, FUN=sum, na.rm=TRUE))/(tapply(as.numeric(CMARexport$TradeValue), CMARexport$Year, FUN=sum, na.rm=TRUE))

[1] 0.5907771

[1] 6694680

2011    2012    2013    2014    2015    2016 
3805258 5452911 6889127 7908262 7889383 8223138

2011      2012      2013      2014      2015      2016 
0.5914756 0.6656234 0.7232807 0.7258916 0.5762940 0.4270205

According to U.N. data, Costa Rica exported an average of 6,694,679.8 USD of shark products per year between 2011 and 2016. That's 59 percent of total exports (as measured in declared value) in the region. However, there has been a steady increase in declared value of exports since 2011 (with the exception of a small dip from 2014 to 2015 and then again from 2015 to 2016). Costa Rican exports accounted for between 42.7 and 72.5 percent of regional shark product exports in any given year. 

In [13]:
#2.3.2 Price per kilo by year of Costa Rica shark exports
(tapply(as.numeric(crexports$TradeValue), crexports$Year, FUN=sum, na.rm=TRUE))/(tapply(as.numeric(crexports$Netweight), crexports$Year, FUN=sum, na.rm=TRUE))

2011      2012      2013      2014      2015      2016 
 1.205432  1.458117  2.560341  2.657748  4.498982 11.775905

Costa Rica has seen what appears to be exponetial increase in price per kilo between 2011 and 2016. The price increased more than $10. This could indicate more fins, or a higher price per fin, or a higher price for shark products in general. 

In [14]:
#2.3.3 Export commodity codes used by Costa Rica for shark products
table(crexports$CommodityCode)


30265 30281 30375 30381 30571 
    3     2     8    33     7 

Number under each HS code represtents the number of times in the given timeframe that each code was used. The UN Defines the HS codes used by Costa Rica as follows:

30265: Dogfish & other sharks, fresh/chilled (excl. fillets/other fish meat of 03.04/livers & roes)

30281: Fish and crustaceans, molluscs and other aquatic invertebrates // Fish, fresh or chilled, excluding fish fillets and other fish meat of heading 03.04. // - Other fish, excluding livers and roes : // -- Dogfish and other sharks

30375: Dogfish & other sharks, frozen (excl. fillets/other fish meat of 03.04/livers & roes)

30381: Fish and crustaceans, molluscs and other aquatic invertebrates // Fish, frozen, excluding fish fillets and other fish meat of heading03.04. // - Other fish, excluding livers and roes : // -- Dogfish and other sharks

30571: Fish and crustaceans, molluscs and other aquatic invertebrates // Fish, dried, salted or in brine; smoked fish, whether or not cooked before or during the smoking process; flours, meals and pellets of fish, fit for human consumption. // - Fish fins, heads, tails, maws and other edible fish offal : // -- Shark fins

This shows just how vague the commody classification system is in Costa Rica when it comes to shark products. It is unclear how much of this is actually shark.

### 2.4 El Salvador Exports

It is unusual for a Northern triangle country to be as big a shark exporter as El Salvador appeared to be. Here we will do a basic analysis of Salvadoran shark exports in order to assist our understanding of it's role. Anecdotly, El Salvador became an important offload port of Wang vessels around 2015 when the vessels stopped offloading at Costa Rican port. They apparently also offloaded in El Salvador some years earlier, left, and then came back, according to port officials in Puerto Corsain, El Salvador.

In [15]:
#2.4.1 Create El Salvador subset, average value of shark exports per year, and shark export value per year in order of least to most.
esexports = filter(CMAR, TradeFlow == 'Export', Reporter == 'El Salvador')
sum(esexports$TradeValue)/6
sort(tapply(as.numeric(esexports$TradeValue), esexports$Year, FUN=sum, na.rm=TRUE))

[1] 1015937

2013    2014    2016    2012    2015 
 525969  561068 1513669 1688434 1806481

El Salvador has an inconsistent export record when it comes to shark commodity exports. In 2015 and 2016, it exported nearly 1.5 million USD of shark products. The only other year similar was 2012. This supports the idea that there was a large influx of shark offloads in those years.

In [16]:
#2.4.2 Price per kilo by year of El Salvador shark exports
(tapply(as.numeric(esexports$TradeValue), esexports$Year, FUN=sum, na.rm=TRUE))/(tapply(as.numeric(esexports$Netweight), esexports$Year, FUN=sum, na.rm=TRUE))

2012      2013      2014      2015      2016 
76.399729 55.847207 48.413841  1.919481  3.235303

The price per kilo of shark products in 2012 in El Salvador is consistent with the price of shark fins (see section 3.1.5 and 3.1.6). The price in 2016 is about three times the normal price of shark meat, indicating a mix.

In [17]:
#2.4.3 Export commodity codes used by El Salvador for shark products
table(esexports$CommodityCode)


30381 30571 
    5     7 

30381: Fish and crustaceans, molluscs and other aquatic invertebrates // Fish, frozen, excluding fish fillets and other fish meat of heading03.04. // - Other fish, excluding livers and roes : // -- Dogfish and other sharks

30571: Fish and crustaceans, molluscs and other aquatic invertebrates // Fish, dried, salted or in brine; smoked fish, whether or not cooked before or during the smoking process; flours, meals and pellets of fish, fit for human consumption. // - Fish fins, heads, tails, maws and other edible fish offal : // -- Shark fins


In [18]:
#2.4.4 Recipients of Salvadoran shark exports and the value of exports to each receiving country.

sort(table(esexports$Partner))
sort(tapply(as.numeric(esexports$TradeValue), esexports$Partner, FUN=sum, na.rm=TRUE))


          Areas, nes            Argentina             Barbados 
                   0                    0                    0 
              Belize               Brazil                China 
                   0                    0                    0 
            Colombia                 Cuba              Ecuador 
                   0                    0                    0 
         El Salvador            Guatemala             Honduras 
                   0                    0                    0 
             Iceland            Indonesia                Japan 
                   0                    0                    0 
            Malaysia            Nicaragua               Panama 
                   0                    0                    0 
                Peru             Portugal   Russian Federation 
                   0                    0                    0 
               Spain            Sri Lanka  Trinidad and Tobago 
                   0                   

Costa Rica               Mexico      Other Asia, nes 
               83770               173200              1241265 
               World China, Hong Kong SAR 
             1501011              3096375

Hong Kong and other asian ports recieve the majority of Salvadoran shark exports. The 173,200 USD to Mexico. And, based on interviews with experts in the industry, Costa Rica importing 83,770 USD of shark products is probably for processing prior to re-export. 

In [19]:
# 2.4.5 Percentage of total CMAR shark exports acccounted for by Salvadoran exports per year compared to Costa Rica, below.
f = filter(CMARexport, Year%in% c("2012","2013","2014","2015","2016"))
(tapply(as.numeric(esexports$TradeValue), esexports$Year, FUN=sum, na.rm=TRUE))/(tapply(as.numeric(f$TradeValue), f$Year, FUN=sum, na.rm=TRUE))

(tapply(as.numeric(crexports$TradeValue), crexports$Year, FUN=sum, na.rm=TRUE))/(tapply(as.numeric(CMARexport$TradeValue), CMARexport$Year, FUN=sum, na.rm=TRUE))

2012       2013       2014       2015       2016 
0.20610296 0.05522082 0.05149988 0.13195760 0.07860352

2011      2012      2013      2014      2015      2016 
0.5914756 0.6656234 0.7232807 0.7258916 0.5762940 0.4270205

Salvadoran exports in terms of declared value make up between 5 and 20 percent of total CMAR shark product exports. In 2012, Salvadoran exports were responsible for 20.6 percent of regional exports. In the same year Costa Rica was responsible for 66.5 percent of regional exports. In 2015, El Salvador accounted for 13 percent of regional shark product exports and Costa Rica for 57.6 percent of exports. Those numbers dropped for both countries in 2016.

## 3. COSTA RICA SHARK EXPORTS (ImportGenius)

In this section, using Import Genius trade data, we will take a closer look at Costa Rican shark exports and the companies most involved. 

In [20]:
head(CR)

PRODUCT_DESCRIPTION                        ProductDescription2              
1 CARCHARHINUS FALCIFORMIS DRYED SHARK FINS- CARCHARHINUS FALCIFORMIS-        
2 CARCHARHINUS FALCIFORMIS DRYED SHARK FINS- CARCHARHINUS FALCIFORMIS-        
3 CARCHARHINUS FALCIFORMIS DRYED SHARK FINS- CARCHARHINUS FALCIFORMIS-        
4 CARCHARHINUS FALCIFORMIS DRYED SHARK FINS- CARCHARHINUS FALCIFORMIS-        
5 CARCHARHINUS FALCIFORMIS DRYED SHARK FINS- CARCHARHINUS FALCIFORMIS-        
6 PRIONACE GLAUCA DRYED SHARK FINS-          PRIONACE GLAUCA DRYED SHARK FINS-
  PRODUCT SHIPPER               EXPORTCOMPANY                      
1 aleta   INVERSIONES CRUZ Z SA INVERSIONES CRUZ Z SOCIEDAD ANONIMA
2 aleta   INVERSIONES CRUZ Z SA INVERSIONES CRUZ Z SOCIEDAD ANONIMA
3 aleta   INVERSIONES CRUZ Z SA INVERSIONES CRUZ Z SOCIEDAD ANONIMA
4 aleta   INVERSIONES CRUZ Z SA INVERSIONES CRUZ Z SOCIEDAD ANONIMA
5 aleta   INVERSIONES CRUZ Z SA INVERSIONES CRUZ Z SOCIEDAD ANONIMA
6 aleta   INVERSIONES CRUZ Z SA INVERSIONES CRUZ Z SOCIEDAD ANONIMA
  SHIPPER_ADDRESS                                                         
1 DE MARISCOS BAR 25 M. AL NORTE Y 35 MTS. AL OESTE EL COCAL DE PUNTARENAS
2 DE MARISCOS BAR 25 M. AL NORTE Y 35 MTS. AL OESTE EL COCAL DE PUNTARENAS
3 DE MARISCOS BAR 25 M. AL NORTE Y 35 MTS. AL OESTE EL COCAL DE PUNTARENAS
4 DE MARISCOS BAR 25 M. AL NORTE Y 35 MTS. AL OESTE EL COCAL DE PUNTARENAS
5 DE MARISCOS BAR 25 M. AL NORTE Y 35 MTS. AL OESTE EL COCAL DE PUNTARENAS
6 DE MARISCOS BAR 25 M. AL NORTE Y 35 MTS. AL OESTE EL COCAL DE PUNTARENAS
  EXPORT_DATE EXPORT_YEAR HS_CODE    BRAND CIF      NET_WEIGHT PACKAGE       
1 8/28/17     2017        3.0571e+10 SM     5266.72  31.40     BULTOS/PACKAGE
2 8/28/17     2017        3.0571e+10 SM    36120.88 271.30     BULTOS/PACKAGE
3 8/28/17     2017        3.0571e+10 SM    21601.68 199.96     BULTOS/PACKAGE
4 8/28/17     2017        3.0571e+10 SM    20084.99 248.30     BULTOS/PACKAGE
5 8/28/17     2017        3.0571e+10 SM     4350.10 106.10     BULTOS/PACKAGE
6 8/28/17     2017        3.0571e+10 SM     4669.60  89.80     BULTOS/PACKAGE
  X.PACKAGES GROSS_WEIGHT NET_WEIGHT2 TOTAL_INVOICE
1 1          1476         1416.16     107116.3     
2 8          1476         1416.16     107116.3     
3 5          1476         1416.16     107116.3     
4 6          1476         1416.16     107116.3     
5 2          1476         1416.16     107116.3     
6 3          1476         1416.16     107116.3     
  REMARKS                                                                                      
1 GUIA AEREA: 020-5720-3775 , REC 4631, EXPORTADOR: INVERSIONES CRUZ. Z. S.A. CED: 310123731211
2 GUIA AEREA: 020-5720-3775 , REC 4631, EXPORTADOR: INVERSIONES CRUZ. Z. S.A. CED: 310123731211
3 GUIA AEREA: 020-5720-3775 , REC 4631, EXPORTADOR: INVERSIONES CRUZ. Z. S.A. CED: 310123731211
4 GUIA AEREA: 020-5720-3775 , REC 4631, EXPORTADOR: INVERSIONES CRUZ. Z. S.A. CED: 310123731211
5 GUIA AEREA: 020-5720-3775 , REC 4631, EXPORTADOR: INVERSIONES CRUZ. Z. S.A. CED: 310123731211
6 GUIA AEREA: 020-5720-3775 , REC 4631, EXPORTADOR: INVERSIONES CRUZ. Z. S.A. CED: 310123731211
  TRANSPORTATION
1 Aereo         
2 Aereo         
3 Aereo         
4 Aereo         
5 Aereo         
6 Aereo

### 3.1 Costa Rica Shark Export Basics

In [21]:
# 3.1.1 Total shark product exports in USD and weight.
sum(CR$CIF)
sum(CR$NET_WEIGHT)

[1] 70708029

[1] 95243637

Costa Rica exported 70,708,029.32 USD in shark products between 2011 and Aug. 2017. That was 95,243,637.38 kilos of shark products. CIF includes the price of commodity plus price of insurance and freight. 

In [22]:
# 3.1.2 Value of exports (USD) divided into fins, meat, bait, and other. 

tapply(CR$CIF, CR$PRODUCT, FUN=sum)

aleta    carnada      carne      other 
47450454.8   163912.6 22797020.2   296641.6

Aleta= Fin
Carnada = Bait
Carne = Meat
Other = Other
Between 2011 and Aug. 2017, Costa Rica exported 47,450,454.84 USD of fins and 22,797,020.24 USD of meat.

In [23]:
#3.1.3 Price per kilo of each product

(tapply(CR$CIF, CR$PRODUCT, FUN=sum))/(tapply(CR$NET_WEIGHT, CR$PRODUCT, FUN=sum))

aleta   carnada     carne     other 
6.3929181 1.6059213 0.2610953 0.7302827

Costa Rica exporters were exporting shark meat at an average just over 0.25 USD per kilo and around 6 USD for fins. But export price was affected by a flood of shark products in 2014 that knocked the price down.

In [24]:
#3.1.4 Import Genius data vs. UN Comtrade data (start by creating subset from 2011-2016 to compare like data)
s = filter(CR, EXPORT_YEAR%in%c("2011","2012","2013","2014","2015","2016"))
(sum(crexports$TradeValue)/6)
(sum(s$CIF)/6)
sum(s$CIF)
sum(as.numeric(s$NET_WEIGHT), na.rm=TRUE)

(sum(crexports$TradeValue))/(sum(s$CIF))

(tapply(crexports$TradeValue, crexports$Year, FUN=sum))
(tapply(s$CIF, s$EXPORT_YEAR, FUN=sum))

(tapply(crexports$TradeValue, crexports$Year, FUN=sum))/(tapply(s$CIF, s$EXPORT_YEAR, FUN=sum))

[1] 6694680

[1] 11469100

[1] 68814600

[1] 95124404

[1] 0.5837145

2011    2012    2013    2014    2015    2016 
3805258 5452911 6889127 7908262 7889383 8223138

2011     2012     2013     2014     2015     2016 
 7999053  4777887  1640621 45167477  5165418  4064145

2011      2012      2013      2014      2015      2016 
0.4757135 1.1412808 4.1990984 0.1750875 1.5273466 2.0233377

UN COMTRADE data only accounted for an average 58 percent of total shark product exports reported by Import Genius in Costa Rica. In 2012, the value of exports was nearly the same. However, in 2016, Import Genius recorded half as much declared value as UN COMTRADE. In 2014, reported only 17.5 percent of the exports reported by Import Genius. Some of this difference could be explained by the fact that CIF also includes insurance and freight. 

In [25]:
#3.1.5 FIN FACTS. Create fin subset and analyze for 1) USD value of fins exported per year 2) Kilos of fins exported per year 3) Price per kilo exported each year.

fins = filter(CR, PRODUCT == 'aleta')
sort(tapply(fins$CIF, fins$EXPORT_YEAR, FUN=sum))
sort(tapply(fins$NET_WEIGHT, fins$EXPORT_YEAR, FUN=sum))
(tapply(fins$CIF, fins$EXPORT_YEAR, FUN=sum))/(tapply(fins$NET_WEIGHT, fins$EXPORT_YEAR, FUN=sum))

2013     2017     2016     2015     2012     2011     2014 
 1110219  1716734  3292720  4214358  4369673  7501978 25244774

2017       2013       2016       2012       2015       2011       2014 
  22654.42   27756.58   43871.14   53987.31   54827.37  102521.46 7116728.45

2011      2012      2013      2014      2015      2016      2017 
73.174705 80.938892 39.998398  3.547244 76.865951 75.054351 75.779200

Costa Rican shark fins generally export between 73 and 80 USD per kilo. However, 2013 and 2014 saw a drop in price. 2014, which saw the most significant drop, also saw over 7 million kilos of shark fins exported, as opposed to the next runner up (2011) which saw 102,000 kilos exported. I assume this surplus of product caused the prices to drastically drop. Still, shark fin exports were worth 25,244,773.72 USD in 2014, more than three times the next most profitable year (2011).

In [26]:
#3.1.6 MEAT FACTS. Create meat subset and analyze for 1) USD value of meat exported per year 2) Kilos of meat exported per year 3) Price per kilo exported each year.

meat = filter(CR, PRODUCT == 'carne')
sort(tapply(meat$CIF, meat$EXPORT_YEAR, FUN=sum))
sort(tapply(meat$NET_WEIGHT, meat$EXPORT_YEAR, FUN=sum))
(tapply(meat$CIF, meat$EXPORT_YEAR, FUN=sum))/(tapply(meat$NET_WEIGHT, meat$EXPORT_YEAR, FUN=sum))

2017       2012       2011       2013       2016       2015       2014 
  176695.1   350314.2   407402.2   474751.8   650331.3   842459.7 19895066.0

2017        2012        2013        2016        2011        2015 
   96578.94   181755.00   195118.00   326666.55   407725.46   508629.00 
       2014 
85596549.00

2011      2012      2013      2014      2015      2016      2017 
0.9992072 1.9273978 2.4331521 0.2324284 1.6563344 1.9908107 1.8295405

Shark meat exported at between 1 and 2.4 USD per kilo every year except for 2014, where price per kilo dropped to $.23. In 2014, 85,596,549 kilos of meat were exported (next runner up, 2015, 508,629 kilos). In 2014, 19,895,066.02 USD of shark products were exported that year (as opposed to 842,459.7 USD exported in 2015). Could the price drop be due to oversupply? (See section 3.3 and 3.4 where two major exporters come on the scene.)

### 3.2 Biggest Costa Rican Export Companies

In [27]:
#3.2.1 List of biggest export companies by weight exported and by value in order from least to most.
sort(tapply(CR$CIF, CR$SHIPPER, FUN=sum))
sort(tapply(CR$NET_WEIGHT, CR$SHIPPER, FUN=sum))

A & E PELAGIDOS DE CENTROAMERICA SOCIEDAD ANONIMA 
                                                1259.90 
                                  EXPORTADORA FRUMAR SA 
                                                3931.90 
                  SIMA LATINOAMERICANA SOCIEDAD ANONIMA 
                                                4680.00 
          INVERSIONES CANO Y CASTRO CR SOCIEDAD ANONIMA 
                                                9436.93 
                                   MUNDIPLANTA LIMITADA 
                                               12880.00 
             COMERCIALIZADORA WANG CWS SOCIEDAD ANONIMA 
                                               17600.00 
                                        -NOT AVAILABLE- 
                                               26875.00 
                              DISTRIBUIDORA MULTIMAR SA 
                                               33319.00 
                                 INVERSIONES TICOMAR SA 
                                               36400.00 
                MARISCOS DE CORAZON MC SOCIEDAD ANONIMA 
                                               53922.68 
                     EXPORTACIONES MARINAS DEL GOLFO SA 
                                               62521.93 
                                 TICO PEZ F.S.T. UNO SA 
                                               91492.00 
                          QUIROS QUIROS DENIS FRANCISCO 
                                              147543.80 
                 INVERSIONES TUCAN I T SOCIEDAD ANONIMA 
                                              271740.00 
MARISCOS OCEANICAS DEL PACIFICO M.O.P.SOCIEDAD ANONIMA. 
                                              409920.00 
                                   INDUSTRIAS MARTEC SA 
                                              483237.60 
                                               SOTOPENA 
                                              491456.74 
                                  PEKO INTERNACIONAL SA 
                                              613344.13 
              COMERCIALIZADORA RIO CUARTO COSTA RICA SA 
                                              748718.00 
                        TIBURON PINTO INTERNANCIONAL SA 
                                              752567.70 
                             TRANSPORTES EL PESCADOR SA 
                                             3145139.90 
  TUNA VERDE IMPOEXPORT DE COSTA RICA, SOCIEDAD ANONIMA 
                                             3626075.00 
                                 SMALLEY DEVELOPMENT SA 
                                             6536390.07 
                                CAGEVY INVERSIONES S.A. 
                                            17058471.00 
                                  INVERSIONES CRUZ Z SA 
                                            36069106.04

A & E PELAGIDOS DE CENTROAMERICA SOCIEDAD ANONIMA 
                                                  10.00 
                  SIMA LATINOAMERICANA SOCIEDAD ANONIMA 
                                                  11.61 
                              DISTRIBUIDORA MULTIMAR SA 
                                                 274.50 
                                  EXPORTADORA FRUMAR SA 
                                                 963.70 
                MARISCOS DE CORAZON MC SOCIEDAD ANONIMA 
                                                6080.13 
                        TIBURON PINTO INTERNANCIONAL SA 
                                                7201.62 
          INVERSIONES CANO Y CASTRO CR SOCIEDAD ANONIMA 
                                                8786.96 
                                        -NOT AVAILABLE- 
                                               21500.00 
             COMERCIALIZADORA WANG CWS SOCIEDAD ANONIMA 
                                               22000.00 
                                 INVERSIONES TICOMAR SA 
                                               26000.00 
                     EXPORTACIONES MARINAS DEL GOLFO SA 
                                               28945.34 
                                   MUNDIPLANTA LIMITADA 
                                               35460.00 
                                 SMALLEY DEVELOPMENT SA 
                                               64223.57 
                                 TICO PEZ F.S.T. UNO SA 
                                               76664.00 
                          QUIROS QUIROS DENIS FRANCISCO 
                                               89317.00 
                                   INDUSTRIAS MARTEC SA 
                                               95463.45 
                 INVERSIONES TUCAN I T SOCIEDAD ANONIMA 
                                              105000.00 
MARISCOS OCEANICAS DEL PACIFICO M.O.P.SOCIEDAD ANONIMA. 
                                              273000.00 
                                               SOTOPENA 
                                              306523.00 
                                  PEKO INTERNACIONAL SA 
                                              424000.00 
              COMERCIALIZADORA RIO CUARTO COSTA RICA SA 
                                              583765.00 
                                  INVERSIONES CRUZ Z SA 
                                             2765305.00 
  TUNA VERDE IMPOEXPORT DE COSTA RICA, SOCIEDAD ANONIMA 
                                             4528820.00 
                             TRANSPORTES EL PESCADOR SA 
                                            21481648.00 
                                CAGEVY INVERSIONES S.A. 
                                            64292674.50

The biggest export companies in Costa Rica by both weight and declared value are Inversiones Cruz Z S.A., Cagevy Inversiones S.A, Transportes El Pescador S.A., and Tuna Verde Impoexport S.A.

Inversiones Cruz is exclusively a fin exporter, and exported 36,069,106.04 USD of shark products over 6.5 years. 

Cagevy Inversiones operates out of Mariscos Wang dock and exports all shark products. It exported 17,058,471 USD over time.

Transportes el Pescador is Fabio Wang's company. It is a managing company for a large shark fishing fleet. It exported 3,145,139.9 USD in shark products. 


## 3.3 Transportes el Pescador

In [28]:
#3.3.1 Create Transportes el Pescador Subset and analyse for export value and weight, and then list per year.

TeP = filter(CR, SHIPPER == 'TRANSPORTES EL PESCADOR SA')
sum(TeP$CIF)
tapply(TeP$CIF, TeP$EXPORT_YEAR, FUN=sum)
sum(TeP$NET_WEIGHT)
tapply(TeP$NET_WEIGHT, TeP$EXPORT_YEAR, FUN=sum)

[1] 3145140

2014      2015 
3049842.4   95297.5

[1] 21481648

2014     2015 
21421872    59776

Transportes el Pescador only exported for two years-- 2014 and 2015-- with the vast majority of exports in 2014. Transportes el Pescador exported 3,145,139.9 USD of shark product total, with 3,049,842.4 USD of that exported in 2014. That was a total of 21,481,648 kilos of shark products.

In [29]:
#3.3.2 Transportes el Pescador type of product, and price per kilo by product
tapply(TeP$CIF, TeP$PRODUCT, FUN=sum)
(tapply(TeP$CIF, TeP$PRODUCT, FUN=sum))/(tapply(TeP$NET_WEIGHT, TeP$PRODUCT, FUN=sum))

aleta   carne 
  14966 3130174

aleta     carne 
3.5000000 0.1457429

The vast majority (over 3 million USD) of Transportes El Pescador's exports were shark meat not fins.  

## 3.4 Cagevy Inversiones

In [30]:
#3.4.1 Create Transportes el Pescador Subset and analyse for export value and weight, and then list per year.

cagevy = filter(CR, SHIPPER == 'CAGEVY INVERSIONES S.A.')
sum(cagevy$CIF)
sort(tapply(cagevy$CIF, cagevy$EXPORT_YEAR, FUN=sum))
sum(cagevy$NET_WEIGHT)
sort(tapply(cagevy$NET_WEIGHT, cagevy$EXPORT_YEAR, FUN=sum))

[1] 17058471

2017     2016     2015     2014 
   23000   130921   200500 16704050

[1] 64292674

2017       2016       2015       2014 
   23000.0    59674.5    96000.0 64114000.0

Cagevy Inversiones first came on the scene in 2014. Since then it exported 17,058,471 USD of shark products, which weighed 64,292,674.5 kilos. The biggest year for Cagevy was 2014 and exports have decreased drastically since then. Cagevy exported 16,704,050 USD of shark product in 2014, which weighed 64,114,000. This is the vast majority of Cagevy exports.

In [31]:
#3.4.2 Cagevy type of product, and price per kilo by product
tapply(cagevy$CIF, cagevy$PRODUCT, FUN=sum)
(tapply(cagevy$CIF, cagevy$PRODUCT, FUN=sum))/(tapply(cagevy$NET_WEIGHT, cagevy$PRODUCT, FUN=sum))

carne 
17058471

carne 
0.2653253

Cagevy has only ever exported shark meat. The average price per kilo of the export was $.27. 

### 3.5 Inversiones Cruz Z S.A.

In [32]:
#3.5.1 Create Inversiones Cruz subset and analyze for export value and weight, and then list per year.

ic = filter(CR, SHIPPER == 'INVERSIONES CRUZ Z SA')
sum(ic$CIF)
sort(tapply(ic$CIF, ic$EXPORT_YEAR, FUN=sum))
sum(ic$NET_WEIGHT)
sort(tapply(ic$NET_WEIGHT, ic$EXPORT_YEAR, FUN=sum))

[1] 36069106

2013     2017     2012     2016     2015     2011     2014 
 1026061  1710368  2503633  2987941  3078999  3627515 21134589

[1] 2765305

2013       2017       2012       2011       2015       2016       2014 
  12986.58   22199.81   24035.62   26919.20   27840.22   33147.03 2618176.54

In 6.5 years, (2011-Aug.2017) Inversiones Cruz exported 36,069,106.04 USD of shark products, weighing 2,765,305 kilos. Their biggest year was also 2014 in which they exported 21,124,589.44 USD of shark products.  

In [33]:
#3.5.2 Inversiones Cruz type of product, and price per kilo by product
tapply(ic$CIF, ic$PRODUCT, FUN=sum)
(tapply(ic$CIF, ic$PRODUCT, FUN=sum))/(tapply(ic$NET_WEIGHT, ic$PRODUCT, FUN=sum))

aleta 
36069106

aleta 
13.04345

Inversiones Cruz only exports fins. On an average, it exports fins at about $13 per kilo. However, the 2014 anomaly accounts for that low price.

In [34]:
#3.5.3 Inversiones Cruze price per kilo of shark fins
(tapply(ic$CIF, ic$EXPORT_YEAR, FUN=sum))/(tapply(ic$NET_WEIGHT, ic$EXPORT_YEAR, FUN=sum))

2011       2012       2013       2014       2015       2016       2017 
134.755674 104.163461  79.009349   8.072255 110.595344  90.142031  77.044252

Inversiones Cruz exported shark fins from between 134 USD and 77 USD over the years. However, in 2014, the price dropped to just over 8 USD per kilo. (Remember that was the year that they exported the most product and brought in the most revenue)

## 4. COSTA RICA OFFLOADS

In this section, using offload data provided by INCOPESCA, we will look at the amount of sharks offloaded in Costa Rica vs. other species, identify trends, and determine whether or not there are fleets specifically dedicated to shark fishing.

In [35]:
head(offload)

Fecha    Ano  Bandera    Puerto     Muelle     Solicitud Desembarque   
1 1-Jan-16 2016 Costa Rica Puntarenas Palmarenos 3464      Aleta Adherida
2 1-Jan-16 2016 Costa Rica Puntarenas Palmarenos 3464      Aleta Adherida
3 1-Jan-16 2016 Costa Rica Puntarenas Palmarenos 3464      Aleta Adherida
4 1-Jan-16 2016 Costa Rica Puntarenas Palmarenos 3464      Aleta Adherida
5 1-Jan-16 2016 Costa Rica Puntarenas Palmarenos 3464      Aleta Adherida
6 1-Jan-16 2016 Costa Rica Puntarenas Palmarenos 3464      Aleta Adherida
  Aprobado Especies        Species cuerpos KgCuerpos KgAletas Peso Embarcacion
1 Aprobado Marlin Rosado   pescado 18                         404             
2 Aprobado Vela            pescado  8                         196             
3 Aprobado Atun A Amarilla pescado  3                         82              
4 Aprobado Espada          pescado  2                         37              
5 Aprobado Martillo Rosado tiburon  1      5.5       0.4      5.9             
6 Aprobado Zorro Pelagico  tiburon 29      761       36       797

### 4.1 Costa Rica Fish Offload Basics

In [36]:
#4.1.1 Total offloads by species (simplified) in weight and number of bodies.
sort(tapply(as.numeric(offload$Peso), offload$Species, FUN=sum, na.rm=TRUE))
sort(tapply(as.numeric(offload$cuerpos), offload$Species, FUN=sum, na.rm=TRUE))

Warning message in tapply(as.numeric(offload$Peso), offload$Species, FUN = sum, :
“NAs introduced by coercion”

other     pescado     tiburon 
   49762.98 15003028.90 25015482.74

other tiburon pescado 
  24184 1508494 2012911

Between 2011 and 2016, at least 1,508,494 sharks were offloaded at Costa Rica ports, weighing over 25,015,482.736 kilos. That is compared to 2,012,911 fish offloaded in the same period, weighing 15,003,028.9. 

In [37]:
#4.1.2 Size of sharks compared to other fish offloaded.
(tapply(as.numeric(offload$Peso), offload$Species, FUN=sum, na.rm=TRUE))/(tapply(as.numeric(offload$cuerpos), offload$Species, FUN=sum, na.rm=TRUE))

Warning message in tapply(as.numeric(offload$Peso), offload$Species, FUN = sum, :
“NAs introduced by coercion”

other   pescado   tiburon 
 2.057682  7.453399 16.583084

In Costa Rica, the average shark body offloaded weighted 16.6 kilos, as compared to fish offloads which weighed 7.5 kilos per body (think tuna, sea bass etc.)

### 4.2 Shark Offloads

In [38]:
#4.2.1 CREATE subset shark and determine percentage of total catch in weight, and then number of bodies.
tiburon = filter(offload, Species == 'tiburon')
(sum(as.numeric(tiburon$Peso), na.rm=TRUE))/(sum(as.numeric(offload$Peso), na.rm=TRUE))
(sum(as.numeric(tiburon$cuerpos), na.rm=TRUE))/(sum(as.numeric(offload$cuerpos), na.rm=TRUE))

Warning message in eval(expr, envir, enclos):
“NAs introduced by coercion”

[1] 0.6243214

[1] 0.4254565

Sharks made up 62% of total offloads by weight in Costa Rica between 2011 and 2016. Almost 43% of total bodies offloaded at Costa Rican ports were sharks.

In [39]:
#4.2.2 Sharks offloaded per year by number of bodies and weight and percentage of total by weight.
sort(tapply(as.numeric(tiburon$cuerpos), tiburon$Ano, FUN=sum, na.rm=TRUE))

sort(tapply(as.numeric(tiburon$Peso), tiburon$Ano, FUN=sum, na.rm=TRUE))

(tapply(as.numeric(tiburon$Peso), tiburon$Ano, FUN=sum, na.rm=TRUE))/(tapply(as.numeric(offload$Peso), offload$Ano, FUN=sum, na.rm=TRUE))

2015   2016   2013   2014   2012   2011 
128553 136757 227301 281788 296557 437538

2016    2015    2013    2014    2012    2011 
2656900 2901008 3662364 4245261 4927411 6622539

Warning message in tapply(as.numeric(offload$Peso), offload$Ano, FUN = sum, na.rm = TRUE):
“NAs introduced by coercion”

2011      2012      2013      2014      2015      2016 
0.7405156 0.9974474 0.5775360 0.3795852 0.6655843 0.6177098

There as been a decrease in shark offloads in Costa Rica since 2011 in which 437,538 sharks were offloaded, to 2016, where 136,757 sharks were offloaded. Each year sharks made up between 55 and 75 percent of offloads, except for 2014 when shark offloads dropped to under 40 percent and 2012 when sharks were 99 percent of recorded offloads. Neither of these numbers make sense. Sharks would never be nearly 100 percent of offloads which makes me assume there is a data error. And less than 40 percent is low in a year where all of the companies made the most money, and exported the most shark products.

Shark product (by weight kg) offloaded annually in Costa Rica. 
2011: 6,622,538.546 kg (74% of total offload)
2012: 4,927,411.356 kg (99% of total offload)
2013: 3.662,364.36 kg (58% of total offload)
2014: 4.245,261.373 kg (38% of total offload)
2015: 2,901,007.55 kg (67% of total offload)
2016: 2,656,899.551 kg (62% of total offload)


In [40]:
#4.2.2.1 Check the 2012 irregularity.
twelve = filter(offload, Ano==2012)
tapply(as.numeric(twelve$Peso), twelve$Species, FUN=sum, na.rm=TRUE)
4927411.356/(sum(as.numeric(twelve$Peso), na.rm=TRUE))

other    pescado    tiburon 
   4185.76    8423.99 4927411.36

[1] 0.9974474

In [41]:
#4.2.3 Size (measured in kg) of sharks brought in each year. 
(tapply(as.numeric(tiburon$Peso), tiburon$Ano, FUN=sum, na.rm=TRUE))/(tapply(as.numeric(tiburon$cuerpos), tiburon$Ano, FUN=sum, na.rm=TRUE))

2011     2012     2013     2014     2015     2016 
15.13592 16.61539 16.11240 15.06544 22.56663 19.42789

Sharks were heaviest in 2015 where the average offloaded body weighed about 22.6 kilos. The smallest was about 15 kilos. This is the weight of a frozen shark, which generally has already been gutted and the head removed.

### 4.3 Offload Vessels

The vessel names are from years 2014 and 2015 only (due to clerical error), however, all other information is available for all years between 2011 and 2016 including flag state. 

In [42]:
#4.3 A list of all vessels that offloaded in Costa Rica. 
print(sort(table(offload$Embarcacion)))


                MARIA CRISTINA                     Merceditas 
                             1                              1 
                    NEPTUNO XX                          NICHO 
                             1                              1 
                        SHADAY                YAISEL PATRICIA 
                             1                              1 
                     ALETA VII                          ALI I 
                             2                              2 
                       BRIGITH                  CAPITAN PEDRO 
                             2                              2 
                      CATALINA                    COOPEPEZ 19 
                             2                              2 
                      ISAURA I                    LA ROSA III 
                             2                              2 
        MARIA DE LOS ANGELES I                   Rey de Reyes 
                             2                        

In [43]:
#4.3.1 A list of vessels that offloaded shark
print(sort(table(tiburon$Embarcacion)))



                      ANDREY J                       CAP PANA 
                             1                              1 
                   CAPITAN LYN                 CAPITAN RONALD 
                             1                              1 
                        COMBOY COOPEPES V (LICENCIA INACTIVA) 
                             1                              1 
                   COSTA MAR I                         DUKEBO 
                             1                              1 
                    EL TRACHER                     ESPADA III 
                             1                              1 
                        FRUMAR                      Jan Angel 
                             1                              1 
                       LA LIBI                      MAR Y MAR 
                             1                              1 
                     MARIANELA                  Marianella II 
                             1                        

In [44]:
#4.3.2 Shark fishing vessels that offloaded more than one percent of average annual shark catch at a time.

(sum(as.numeric(tiburon$cuerpos), na.rm=TRUE))/6
((sum(as.numeric(tiburon$cuerpos), na.rm=TRUE))/6)*.01
biggies = filter(offload, Species == 'tiburon', cuerpos>=2514)
table(biggies$Embarcacion)

[1] 251415.7

[1] 2514.157


                    BAULA XII     DRAGON XII    HUNG CHI FU HUNG CHI FU 33 
             9              1              2              2              2 
       ITAKA I     TARZAN III    ViCTORIA G.     YU LONG 35       YUN LONG 
             1              1              1              2              1 

The average annual shark catch is around 251,416 bodies, so one percent of that would be 2514 bodies. Just a handfull of vessels offloading more than that amount of a single species of shark at a time.(NOTE: One problem with this test is that each row is for a single species, and multiple rows could represent the same offload. However, that would make the offload bigger, not smaller and does not change the fact that these are some of the biggest shark boats in the game.) 

In [45]:
#4.3.2.1 A visual representation of the offloads by the "biggies"
arrange(biggies,Peso)

Fecha     Ano  Bandera    Puerto     Muelle            Solicitud
1  15-Dec-14 2014 Costa Rica Puntarenas Pulpo Mar         22222    
2  1-Jun-11  2011 Costa Rica Puntarenas Muelle J y Cheng   6913    
3  3-May-14  2014 Belice     Puntarenas INCOPESCA         20666    
4  22-Dec-14 2014 Costa Rica Puntarenas Terminal Pesquera 22298    
5  21-Mar-12 2012 Belice     Puntarenas INCOPESCA         10660    
6  18-Jan-14 2014 Belice     Puntarenas INCOPESCA         19855    
7  30-Sep-14 2014 Belice     Puntarenas INCOPESCA         21710    
8  11-Jun-12 2012 Costa Rica Puntarenas PMT               12018    
9  30-Dec-14 2014 Belice     Puntarenas INCOPESCA         22374    
10 30-Dec-14 2014 Belice     Puntarenas Terminal Pesquera 22374    
11 3-Dec-14  2014 Belice     Puntarenas INCOPESCA         22166    
12 21-Nov-14 2014 Belice     Puntarenas INCOPESCA         22061    
13 17-Apr-13 2013 Belice     Puntarenas INCOPESCA         16391    
14 14-Dec-12 2012 Belice     Puntarenas INCOPESCA         14692    
15 17-Dec-15 2015 Costa Rica Puntarenas Mariscos Wang      3358    
16 8-Jun-11  2011 Belice     Puntarenas Terminal Pesquera  8656    
17 7-Nov-14  2014 Belice     Puntarenas INCOPESCA         21960    
18 7-Dec-11  2011 Belice     Puntarenas INCOPESCA          2799    
19 2-Mar-13  2013 Belice     Puntarenas INCOPESCA         15793    
20 19-Dec-11 2011 Belice     Puntarenas INCOPESCA          2877    
21 27-Feb-14 2014 Belice     Puntarenas INCOPESCA         20076    
22 4-Feb-14  2014 Belice     Puntarenas INCOPESCA         19971    
   Desembarque    Aprobado Especies Species cuerpos KgCuerpos KgAletas Peso   
1  T22233         Aprobado azul     tiburon 3621    207.3     0        207.3  
2  Aleta Adherida Aprobado Gris     tiburon 2734    21,945.00 2,068.00 24013  
3  Aleta Adherida Aprobado azul     tiburon 3802    25116     2850     27966  
4  Aleta Adherida Aprobado Gris     tiburon 6178    381.9     0        381.9  
5  Aleta Adherida Aprobado Gris     tiburon 2900    36,086.00 3,067.00 39153  
6  Aleta Adherida Aprobado azul     tiburon 7994    3971      314      4285   
7  Aleta Adherida Aprobado Gris     tiburon 2537    558.35    4251     4809.35
8  Aleta Adherida Aprobado Gris     tiburon 2732    4,895.30  0        4895.3 
9  Aleta Adherida Aprobado Gris     tiburon 2755    46865     3405     50270  
10 Aleta Adherida Aprobado Gris     tiburon 2755    46,865.00 3,405.00 50270  
11 Aleta Adherida Aprobado Gris     tiburon 3111    46672     3716     50388  
12 Aleta Adherida Aprobado Gris     tiburon 3499    49802     3821     53623  
13 Aleta Adherida Aprobado Gris     tiburon 3131    50,108.00 4,776.00 54884  
14 Aleta Adherida Aprobado Gris     tiburon 2555    51,113.00 4,602.00 55715  
15 Aleta Adherida Aprobado Gris     tiburon 2750    59,455.00 0        59455  
16 Aleta Adherida Aprobado Gris     tiburon 4600    64,557.00 4,850.00 69407  
17 Aleta Adherida Aprobado Gris     tiburon 3269    65229     4977     70206  
18 Aleta Adherida Aprobado Gris     tiburon 4747    70,028.00 5,862.00 75890  
19 Aleta Adherida Aprobado Gris     tiburon 9255    74,037.00 4,917.00 78954  
20 Aleta Adherida Aprobado Gris     tiburon 3975    75,520.00 6,283.00 81803  
21 Aleta Adherida Aprobado Gris     tiburon 5519    77272     6039     83311  
22 Aleta Adherida Aprobado Gris     tiburon 7097    85167     6600     91767  
   Embarcacion   
1  BAULA XII     
2                
3  DRAGON XII    
4  ITAKA I       
5                
6  YUN LONG      
7  HUNG CHI FU   
8                
9  YU LONG 35    
10 YU LONG 35    
11 TARZAN III    
12 DRAGON XII    
13               
14               
15 ViCTORIA G.   
16               
17 HUNG CHI FU 33
18               
19               
20               
21 HUNG CHI FU 33
22 HUNG CHI FU

In [46]:
#4.3.2.2 Number of biggest single-species offloads
print(biggies)

       Fecha  Ano    Bandera     Puerto            Muelle Solicitud
1  21-Mar-12 2012     Belice Puntarenas         INCOPESCA     10660
2  11-Jun-12 2012 Costa Rica Puntarenas               PMT     12018
3  14-Dec-12 2012     Belice Puntarenas         INCOPESCA     14692
4   1-Jun-11 2011 Costa Rica Puntarenas  Muelle J y Cheng      6913
5   8-Jun-11 2011     Belice Puntarenas Terminal Pesquera      8656
6   7-Dec-11 2011     Belice Puntarenas         INCOPESCA      2799
7  19-Dec-11 2011     Belice Puntarenas         INCOPESCA      2877
8   2-Mar-13 2013     Belice Puntarenas         INCOPESCA     15793
9  17-Apr-13 2013     Belice Puntarenas         INCOPESCA     16391
10 18-Jan-14 2014     Belice Puntarenas         INCOPESCA     19855
11  4-Feb-14 2014     Belice Puntarenas         INCOPESCA     19971
12 27-Feb-14 2014     Belice Puntarenas         INCOPESCA     20076
13  3-May-14 2014     Belice Puntarenas         INCOPESCA     20666
14 30-Sep-14 2014     Belice Puntarenas         

In [47]:
#4.3.3 Number of shark bodies offloaded in single-species offloads, and total weight of catch offloaded by biggies, and percentage of all offloads.
sum(as.numeric(biggies$cuerpos))
(sum(as.numeric(biggies$cuerpos)))/22
(sum(as.numeric(biggies$cuerpos)))/(sum(as.numeric(tiburon$cuerpos), na.rm=TRUE))
sum(as.numeric(biggies$Peso))
(sum(as.numeric(biggies$Peso)))/22
(sum(as.numeric(biggies$Peso)))/(sum(as.numeric(tiburon$Peso), na.rm=TRUE))

[1] 91516

[1] 4159.818

[1] 0.06066713

[1] 1031654

[1] 46893.36

[1] 0.04124061

In six years, these "biggies" offloaded 91,516 sharks in the biggest single-species shark offloads (an average of 4,160 sharks per offload, compared to an overall average of 59 sharks [4.3.3.1]). That means just 22 single species offloads accounted for 6% of the total shark offloads (as measured in bodies offload) in Costa Rica over that period. It was 4% of the total weight.  (Note: it is actually more, in the period in which most of these boats were active. See section 4.4) 

In [48]:
#4.3.3.1 The average number of sharks offloaded in a single species offload
dim(tiburon)
(sum(as.numeric(tiburon$cuerpos),na.rm=TRUE))/25575

[1] 25575    15

[1] 58.98315

In [49]:
#4.3.4 Number of "biggie" offloads by year. 
sort(table(biggies$Ano))


2015 2013 2012 2011 2014 
   1    2    3    4   12 

2014 saw the highest propensity of large offloads. Every other year only had a handful. 

In [50]:
#4.3.4.1 "Biggie" offloads by year by vessel.
table(biggies$Embarcacion, biggies$Ano)

                
                 2011 2012 2013 2014 2015
                    4    3    2    0    0
  BAULA XII         0    0    0    1    0
  DRAGON XII        0    0    0    2    0
  HUNG CHI FU       0    0    0    2    0
  HUNG CHI FU 33    0    0    0    2    0
  ITAKA I           0    0    0    1    0
  TARZAN III        0    0    0    1    0
  ViCTORIA G.       0    0    0    0    1
  YU LONG 35        0    0    0    2    0
  YUN LONG          0    0    0    1    0

The Baula XIII was responsible for all of the biggest single-species offloads between 2011 and 2013. In 2014, the year of the most big offloads, the DRAGON XII, HUNG CHI FU, HUNG CHI FU 33, ITAKA I, TARZAN III, YU LONG 35, and the YUN LONG, were responsible for the biggest single-species offloads. All but the ITAKA I is registered to, or managed by, a company associated with Fabio Wang. In 2015, the only "biggie" offload was by the Victoria G, which offloaded at Mariscos Wang (Fabio Wang's dock), where its owner rents space. 

In [51]:
#4.3.4.2 Analysis of 2014 "biggie" offloads

x = filter(biggies, Ano == 2014)
y = filter(tiburon, Ano == 2014)
sum(as.numeric(x$cuerpos), na.rm=TRUE)
(sum(as.numeric(x$cuerpos), na.rm=TRUE))/12
(sum(as.numeric(x$cuerpos), na.rm=TRUE))/(sum(as.numeric(y$cuerpos), na.rm=TRUE))

sum(as.numeric(x$Peso), na.rm=TRUE)
(sum(as.numeric(x$Peso), na.rm=TRUE))/12
(sum(as.numeric(x$Peso), na.rm=TRUE))/(sum(as.numeric(y$Peso), na.rm=TRUE))

[1] 52137

[1] 4344.75

[1] 0.1850221

[1] 487484.5

[1] 40623.71

[1] 0.1148303

In 2014, the 12 biggest single species shark offloads accounted for 52,137 shark bodies, or 18.5% of total shark bodies offloaded that year. The offloads were an average of 4,345 sharks per offload. Each offload was an average of just over 20 metric tons of shark.

In [52]:
#4.3.5 Flag states of biggie vessels.

table(biggies$Bandera)
table(biggies$Embarcacion, biggies$Bandera)


    Belice Costa Rica 
        17          5 

                
                 Belice Costa Rica
                      7          2
  BAULA XII           0          1
  DRAGON XII          2          0
  HUNG CHI FU         2          0
  HUNG CHI FU 33      2          0
  ITAKA I             0          1
  TARZAN III          1          0
  ViCTORIA G.         0          1
  YU LONG 35          2          0
  YUN LONG            1          0

Seventeen of the total 22 "biggie" were done by Belize flagged vessels, the same ones responsible for the biggie offloads in 2014. 

In [53]:
#4.3.6 Shark product offloads (by weight and body count) per flag state
sort(tapply(as.numeric(tiburon$Peso), tiburon$Bandera, FUN=sum, na.rm=TRUE))
4931733.216/(sum(as.numeric(tiburon$Peso), na.rm=TRUE))
sort(tapply(as.numeric(tiburon$cuerpos), tiburon$Bandera, FUN=sum, na.rm=TRUE))
247431/(sum(as.numeric(tiburon$cuerpos), na.rm=TRUE))

Taiwan  Indonesia                Belice Costa Rica 
   52560.5    84970.0   358414.5  4931733.2 19587804.6

[1] 0.1971472

Taiwan  Indonesia                Belice Costa Rica 
      2132       4794      20973     247431    1233164

[1] 0.1640252

Almost 20 percent of the total sharks offloaded between 2011 and 2016 (16% by weight) were from Belizean flagged vessels. (Note: Belize does not have a Pacific coast, and its flag is generally considered a flag of convenience.) 

### 4.4 Belizean Flagged Vessels

In [54]:
#4.4.1 Create a subset of Belizean flagged vessels and list them.
belize = filter(offload, Bandera == "Belice")
table(belize$Embarcacion)
dim(belize)


                   DRAGON XII    HUNG CHI FU HUNG CHI FU 33     TARZAN III 
           836             10             32             16             10 
       YU LONG     YU LONG 35       YUN LONG 
             6              8              6 

[1] 924  15

There were a total of 924 single-species offloads by Belizean flagged vessels between 2011 and 2016. Of those we have the names of, (from 2014 and 2015), there are the DRAGON XII, HUNG CHI FU, HUNG CHI FU 33, TARZAN III, YU LONG, YU LONG 35, YUN LONG (which is likely a misspelling of YU LONG). All of these vessels are registered to, or managed by, a company associated with Fabio Wang.

In [55]:
#4.4.2 Create a subset of shark offloads by Belizean flagged vessels and compare with total offloads.
bsharks = filter(offload, Bandera == "Belice", Species == "tiburon" )
dim(bsharks)
sum(as.numeric(bsharks$cuerpos))
sum(as.numeric(bsharks$cuerpos))/340
sum(as.numeric(bsharks$Peso))
(sum(as.numeric(bsharks$Peso)))/(sum(as.numeric(belize$Peso), na.rm=TRUE))

[1] 340  15

[1] 247431

[1] 727.7382

[1] 4931733

Warning message in eval(expr, envir, enclos):
“NAs introduced by coercion”

[1] 0.9378966

Belizean flagged vessels offloaded 247,431 shark bodies at Costa Rican ports between 2011 and 2016. (Note: Belizean flagged offloads stopped in 2015, see 4.4.3.) Sharks accounted for almost 94% of offloads (by weight) from Belizean flagged vessels. Of the single-species shark offloads, an average of 728 sharks were offloaded at a time. That is compared to an overall average of 59 sharks [4.3.3.1].

In [56]:
#4.4.3 Shark offloads of Belizean flagged vessel by year (bodies and then weight)
sort(tapply(as.numeric(bsharks$cuerpos), bsharks$Ano, FUN=sum, na.rm=TRUE))
sort(tapply(as.numeric(bsharks$Peso), bsharks$Ano, FUN=sum, na.rm=TRUE))

2013  2014  2012  2011 
50014 56828 57807 82782

2014      2013      2012      2011 
 822564.2 1081640.9 1335251.9 1692276.2

Belizean flagged vessels offloaded at Costa Rican ports through 2014 and then abruptly stopped. However, the offloads did decrease steadily between 2011 and 2014 from 82,782 sharks (2011) to 50,014 (2014). The offloads were from 1,692,276.2 kg (2011) to 822,564.226 kg (2014)

In [57]:
#4.4.3.1 Acutal percentage of sharks offloaded by Belize flagged vessels between 2011 and 2014, and a break down by year.
c = filter(tiburon, Ano%in% c("2011","2012","2013","2014"))
(sum(as.numeric(bsharks$cuerpos),na.rm=TRUE))/(sum(as.numeric(c$cuerpos),na.rm=TRUE))
(tapply(as.numeric(bsharks$cuerpos), bsharks$Ano, FUN=sum, na.rm=TRUE))/(tapply(as.numeric(c$cuerpos), c$Ano, FUN=sum, na.rm=TRUE))
(sum(as.numeric(bsharks$Peso),na.rm=TRUE))/(sum(as.numeric(c$Peso),na.rm=TRUE))
(tapply(as.numeric(bsharks$Peso), bsharks$Ano, FUN=sum, na.rm=TRUE))/(tapply(as.numeric(c$Peso), c$Ano, FUN=sum, na.rm=TRUE))

[1] 0.1990301

2011      2012      2013      2014 
0.1891996 0.1949271 0.2200342 0.2016693

[1] 0.2534608

2011      2012      2013      2014 
0.2555329 0.2709845 0.2953395 0.1937606

In the years Belizean flagged vessels were offloading in Costa Rica, they accounted for between 19 and 22 percent of total sharks offloaded. They accounted for up to 30 percent of shark offloads by weight.

## 5. Conclusions

### 5.1 OVERALL

*Costa Rica is the capital of the shark industry of the Central Tropical Pacific (CMAR)-- it exports more than double the shark products of any other CMAR country. One fleet of Belizean flagged boats is responsible for 20% of the total Costa Rican shark fishing industry. The Wang brothers are the king pins of the shark industry, both in Costa Rica and regionally, and were likely responsible for shifting up to 10% of the total value of the shark regional shark trade from Costa Rica to El Salvador between 2014 and 2015.*

Costa Rica is the capital of the shark fishing industry in the CMAR region--From Ecuador to El Salvador. Over the past six years, Costa Rica was number one exporter of shark products by a large margin, both by both weight and declared value. Costa Rican ports also saw an average of over 200,000 sharks offloaded from fishing vessels per year since 2011. And while the national Costa Rican longline fleet brings in the majority of the sharks, a handfull of large vessels, almost exclusively flagged in Belize, are responsible for the biggest offloads of sharks. Belizean flagged vessels were responsible for nearly 20 percent of total shark offloads in Costa Rica. (In 2014, Fabio's boats were responsible for just under 20 percent of shark offloads alone). However, after 2014, these vessels stopped offloading in Costa Rica. Of the named large shark fishing vessels, the majority belong to, or are associated with Fabio Wang. Fabio Wang's company Transportes el Pescador also made millions off of shark product exports, and was one of the top exporters by volumn from Costa Rica. 

### 5.2 CMAR SHARK TRADE

2. CMAR EXPORTS
    1. Costa Rica is the biggest exporter of shark products in the CMAR region by a large margin.
        1. An average of over USD11.3 million of shark products are exported from CMAR nations annually. [2.1.2]
        2. Of that, 59 percent are from Costa Rica. [2.3.1]
    2. There is reason to believe UN COMTRADE data generally underestimates the shark exports of the region. 
        1. On average, UN COMTRADE data accounted for only 58 percent of shark product exports (measured in declared value) reported by Import Genius in Costa Rica. (Time span of 6 years) [3.1.4]
        2. This could be due to difficulty of identifying the correct commodity codes [2.3.3]
    3. El Salvador has a relatively high declared value of shark exports given the size of the country and lack of registered longliners.
        1.  El Salvador has an inconsistent export record when it comes to shark commodity exports. In the past two years, it exported nearly 1.5 million USD of shark products. The only other year similar was 2012. This supports the idea that there was a large influx of shark offloads in those years. [2.4.1]
        2. In 2015, that was 13.1 percent of regional shark exports [2.4.5] In 2012, El Salvador accounted for 20 percent of regional exports. And in 2014, when Costa Rican exports were at an all time high (72.6 percent of regional exports), El Salvador had its lowest year  (5.5 percent of regional exports). [2.4.5] 
        3. Of note: El Salvador exports some of its shark product to Costa Rica, linking the two nations. [2.4.4]


### 5.3 COSTA RICA SHARK PRODUCT EXPORTS

3. Costa Rica Shark Exports
    1. Between 2011 and 2016, Costa Rica exported USD68.8 million in shark products weighing more than 95 million kilos. [3.1.4] This is more than 1.5 times the USD40.1 million reported by UN Comrade. [2.2.1]
    2. Between 2011 and Aug. 2017, Costa Rica exported USD47.5 million in shark fins, and USD22.8 million in shark meat.
    3. Shark fins export for around USD75 per kilo and mean for USD1.5 per kilo.
        1. Costa Rican shark fins generally export between 73 and 80 USD per kilo. However, 2013 and 2014 saw a drop in price. 2014, which saw the most significant drop, also saw over 7 million kilos of shark fins exported, as opposed to the next runner up (2011) which saw 102,000 kilos exported. I assume this surplus of product caused the prices to drastically drop. Still, shark fin exports were worth 25,244,773.72 USD in 2014, more than three times the next most profitable year (2011). [3.1.5]
        2. Shark meat exported at between 1 and 2.4 USD per kilo every year except for 2014, where price per kilo dropped to USD.23. In 2014, 85,596,549 kilos of meat were exported (next runner up, 2015, 508,629 kilos). In 2014, 19,895,066.02 USD of shark products were exported that year (as opposed to 842,459.7 USD exported in 2015). Could the price drop be due to oversupply? (See section 3.3 and 3.4 where two major exporters come on the scene.) [3.1.6]
    4. The biggest export companies in Costa Rica by both weight and declared value are Inversiones Cruz Z S.A., Cagevy Inversiones S.A, Transportes El Pescador S.A., and Tuna Verde Impoexport S.A. [3.2.1]
        1. Inversiones Cruz is exclusively a fin exporter, and exported 36,069,106.04 USD of shark products over 6.5 years. [3.5.1]
            1. Their biggest year was also 2014 in which they exported 21,124,589.44 USD of shark products.  
            2. Inversiones Cruz only exports fins. [3.5.2]
        2. Cagevy Inversiones operates out of Mariscos Wang dock and exports all shark products. It exported 17,058,471 USD over time. [3.4.1]
            1. Cagevy Inversiones first came on the scene in 2014. Since then it exported 17,058,471 USD of shark products, which weighed 64,292,674.5 kilos. The biggest year for Cagevy was 2014 and exports have decreased drastically since then. Cagevy exported 16,704,050 USD of shark product in 2014, which weighed 64,114,000. This is the vast majority of Cagevy exports.
            2. Cagevy only exports shark meat [3.4.2]
        3. Transportes el Pescador is Fabio Wang's company. It is a managing company for a large shark fishing fleet. It exported 3,145,139.9 USD in shark products.
            1. Transportes el Pescador only exported for two years-- 2014 and 2015-- with the vast majority of exports in 2014. Transportes el Pescador exported 3,145,139.9 USD of shark product total, with 3,049,842.4 USD of that exported in 2014. That was a total of 21,481,648 kilos of shark products. [3.3.1]
            2. Transportes el Pescador mostly exports meat [3.3.2]


### 5.4 COSTA RICA OFFLOADS

4. Costa Rica Offloads
    1. Sharks are a major part of the Costa Rican fishing industry.
        1. Sharks made up an average of 62.4% of total offloads (by weight) in Costa Rica between 2011 and 2016 [4.2.1]
        2. Between 2011 and 2016, at least 1,508,494 sharks were offloaded at Costa Rica ports, weighing over 25,015,482.736 kilos. That is compared to 2,012,911 fish offloaded in the same period, weighing 15,003,028.9. [4.1.1]
        3. The range of sharks offloaded in Costa Rican ports was from 437,538 (2011) to 136,757 (2016) [4.2.2]
            1. 2012 was a bizarre year where almost 100 percent of offloads recorded were sharks. This has to be a data error.
    2. There are vessels that are disproportionately responsible for the shark offloads in Costa Rica
        1. Just a handful of vessels are responsible for the biggest single-species offloads— more than one percent of total annual catch. [4.3.2]
            1. The 22 huge offloads represent 91,516 shark bodies—more than 4,159 sharks at a time on average. [4.3.3] (That’s compared to an average shark offload of 59 shark bodies [4.3.3.1])
            2. That means just 22 single species offloads accounted for 6% of the total shark offloads (as measured in bodies offload) in Costa Rica over that period. [4.3.3]
            3. 2014 saw the highest propensity of large offloads. [4.3.4]
                1.  Fabio Wang’s fleet was responsible for the majority of the biggest offloads. He took over the industry in a major way in 2014.
                    1. In 2014, the year of the most big offloads, the DRAGON XII, HUNG CHI FU, HUNG CHI FU 33, ITAKA I, TARZAN III, YU LONG 35, and the YUN LONG, were responsible for the biggest single-species offloads. All but the ITAKA I is registered to, or managed by, a company associated with Fabio Wang. [4.3.4.1]
                    2. The 2014, large scale offloads accounted for 52,137 sharks or 18.5% of the total shark offloads that year. [4.3.4.2]
                    3. His whole fleet was flagged in Belize. [4.3.5]
    3. Belize flagged vessels are responsible for a significant amount of shark landings.
        1. Belize flagged vessels accounted for 19.7 percent of total shark offloads at Costa Rican ports or 247,431 sharks between 2011 and 2016. [4.3.6]
        2. There were a total of 924 single-species offloads by Belizean flagged vessels between 2011 and 2016. Of those we have the names of, (from 2014 and 2015), there are the DRAGON XII, HUNG CHI FU, HUNG CHI FU 33, TARZAN III, YU LONG, YU LONG 35, YUN LONG (which is likely a misspelling of YU LONG). All of these vessels are registered to, or managed by, a company associated with Fabio Wang. [4.4.4]
        3. Sharks accounted for almost 94% of offloads (by weight) from Belizean flagged vessels. Of the single-species shark offloads, an average of 728 sharks were offloaded at a time. That is compared to an overall average of 59 sharks [4.3.3.1].
        4. Belizean flagged vessels offloaded at Costa Rican ports through 2014 and then abruptly stopped. However, the offloads did decrease steadily between 2011 and 2014 from 82,782 sharks (2011) to 50,014 (2014). The offloads were from 1,692,276.2 kg (2011) to 822,564.226 kg (2014) [4.4.3]
